docs/Guide/application-commands/interaction-handlers/what-are-they.mdx

```diff
@@ -3,7 +3,180 @@ sidebar_position: 1
 title: What are they?
 ---
 
-- Basics
-- Folder structure (pieces related)
+These are interaction handlers! A simple class you can extend to handle almost all the interactions you may receive in your bot, in a simple, clean and abstract way.
 
-<!-- TODO: https://vladfrangu.notion.site/Application-Commands-3d1683f23f0a4fa6b3e682bc8c1cc1b9#ceb1a6f5ec154f85987a95181d512660 -->
+They are located in the `interaction-handlers` folder in your bot (just like how you most likely have a `commands` folder or a `listeners` folder). Same rules apply when it comes to loading and everything!
+
+## How does it work?
+
+When creating your interaction handler, you have to specify the type it is. Currently, there are three types:
+  
+| Enum Name  | What it will run for |
```

Comment:
Prettier formatting for this table where @favna 😢

Reply:
Oh prettier does tables just fine, but seeing as you had other prettier related comments I'm guessing it hadn't run at all when you commented this. Anyway I cleaned it up also:

| Enum Name        | What it will run for                                                                     |
|------------------|------------------------------------------------------------------------------------------|
| Autocomplete     | Autocomplete for interaction commands                                                    |
| Button           | Button presses                                                                           |
| ModalSubmit      | Pop up modals                                                                            |
| MessageComponent | Any message component interactions received (right now buttons, select menus, or modals) |
| SelectMenu       | Drop-down menus in messages                                                              |

https://www.tablesgenerator.com/markdown_tables# is a godsend